show/main.py

```diff
@@ -1670,19 +1670,15 @@ def ssdhealth(device, verbose, vendor):
     options += " -e" if vendor else ""
     run_command(cmd + options, display_cmd=verbose)
 
-# 'fan' subcommand ("show platform fan")
```

Comment:
Is this removed accidentally? Isn't this the reason why the test failed on this PR?

Reply:
please add this back too.